### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by mintxdp** [common-voice/common-voice#4593]
**Describe the bug**
Login/Signup button is unresponsive between 992px to 1199px.

**To Reproduce**
1. Go to https://commonvoice.mozilla.org/en
2. Open Inspect tab in chrome (shortcut : ctrl+shift+i) 
3. Try resizing the window and open login/signup button in right side window.
![mozilla](https://github.com/user-attachments/assets/e759c723-5a61-4a3b-9400-65af5cd1419f)

**Comment by moz-kathyreid** [comment]
Hi there @mintxdp - thank you so much for taking the time to report this bug with the user interface. We've recently made some changes to the design of the Common Voice front end. I can no longer replicate this bug - and am wondering if it is still occurring for you?

**Comment by mintxdp** [comment]
No, the problem seems to be solved but i would say that still it can be improved as it might be less appealing (designwise).
For a reference i have an image, look at this 

![Image](https://github.com/user-attachments/assets/c360aa45-39a4-4f42-bff0-14e5fc17f664)
